public/playerDecisionController.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+export default class playerDecisionController {
+  constructor({ players, generatorCardObject }) {
+    if (!Array.isArray(players)) {
+      throw new Error(
+        `You must provider an array with the players to the playerDecisionController constructor`
+      );
+    }
+
+    this.players = players;
+    this.amountOfPlayers = this.players.length;
+    this.generatorCardObject = generatorCardObject;
+    this.currentPlayerTurn = this.definePlayerTurn();
+
+    this.controlsContainerEle = document.querySelector("#game");
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+
+    this.hit = this.controlsContainerEle.querySelector(".btn--hitting");
+    this.stand = this.controlsContainerEle.querySelector(".btn--standing");
+    this.split = this.controlsContainerEle.querySelector(".btn--splitting");
+    this.double = this.controlsContainerEle.querySelector(".btn--doubling");
+
+    this.hit.addEventListener("click", this.hitting.bind(this));
+    this.stand.addEventListener("click", this.standing.bind(this));
+  }
+
+  controlsHtml() {
+    return `
+ <div class='player-controls'>
+    <button type='button' class='player-controls__btn btn--hitting'> 
+    Hit
+    </button>  
+    <button type='button' class='player-controls__btn btn--standing'> 
+    Stand
+    </button>
+    <button type='button' class='player-controls__btn btn--splitting'> 
+    Split
+    </button>  
+    <button type='button' class='player-controls__btn btn--doubling'> 
+    Double down
+    </button>      
+  </div>`;
+  }
+
+  definePlayerTurn() {
+    let currentPlayerTurn = 0;
+    for (let player of this.players) {
+      if (player.handValue === 21) currentPlayerTurn++;
+      else {
+        document
+          .querySelector(`#${this.players[currentPlayerTurn].name}`)
+          .querySelector(".card")
```

Comment:
You can combine this selector with the previous one like so:
```suggestion
          .querySelector(`#${this.players[currentPlayerTurn].name} .card`)
```